src/web/app/components/question-types/question-statistics/rubric-question-statistics.component.html

```diff
@@ -45,7 +45,7 @@
       </div>
     </div>
     <div class="row">
-      <div id="rubric-recipient-statistics-overall" class="col-sm-12 table-responsive">
+      <div id="rubric-recipient-statistics-overall" class="col-sm-12 table-responsive mb-5">
```

Comment:
May I ask what's the rationale for using `mb-5` here?

Reply:
I forgot to change back after playing around with it 😅